src/commands/create/map.ts

```diff
@@ -0,0 +1,167 @@
+import { flags as oclifFlags } from '@oclif/command';
+import { isValidIdentifier } from '@superfaceai/ast';
+import { SuperJson } from '@superfaceai/one-sdk';
+import { parseDocumentId } from '@superfaceai/parser';
+import { grey, yellow } from 'chalk';
+import { join as joinPath } from 'path';
+
+import {
+  composeUsecaseName,
+  DEFAULT_PROFILE_VERSION_STR,
+  META_FILE,
+} from '../../common';
+import { Command } from '../../common/command.abstract';
+import { developerError, userError } from '../../common/error';
+import { mkdirQuiet } from '../../common/io';
+import { formatShellLog } from '../../common/log';
+import { OutputStream } from '../../common/output-stream';
+import { createMap } from '../../logic/create';
+
+export default class CreateMap extends Command {
+  static strict = false;
+
+  static description =
+    'Creates empty map for specified profile on a local filesystem.';
+
+  static args = [
+    {
+      name: 'profileId',
+      required: true,
+      description: 'Profile Id - scope(optional) and name of a profile',
+    },
+  ];
+
+  static flags = {
+    ...Command.flags,
+    usecase: oclifFlags.string({
+      char: 'u',
+      multiple: true,
+      description: 'Usecases that map contains',
+    }),
+    provider: oclifFlags.string({
+      char: 'p',
+      description: 'Name of a Provider',
+      required: true,
+    }),
+    variant: oclifFlags.string({
+      char: 't',
+      description: 'Variant of a map',
+      dependsOn: ['provider'],
+    }),
+    template: oclifFlags.string({
```

Comment:
I don't really know, this is legacy stuff - I kept it there because we might want to add some new (and actually useful) template. Or I can get rid of it.🤷

Reply:
just get rid of it :shrug: